components/Signin/Signin.js

```diff
@@ -62,94 +77,117 @@ export default class Signin extends Component {
+      // console.log(this._emailInput);
+
+      // says can't read setNativeProps of undefined???!!!
+      // this._emailInput.setNativeProps({ value: '' });
+    }, (error) => { this._sendLogOutAlert() });
   }
 
   render() {
     return (
-      <Container >
+      <Container style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
         <Header />
         <Content style={{ padding: 10 }}>
-          <Form>
-            <Item style={styles.marginBottom} regular>
-              <Input
-                onChangeText={text => this.setState({ email: text })}
-                placeholder="Email"
-                autoCapitalize="none"
-              />
-              {/.+@.+\..+/i.test(this.state.email) && <Icon name={'checkmark-circle'} style={{ color: 'green' }} />}
-            </Item>
-            <Item regular>
-              <Input
-                onChangeText={(text) => { this.setState({ password: text }); }}
-                placeholder="Password"
-                autoCapitalize="none"
-                secureTextEntry
-              />
-              {this.state.password.length >= 6 && <Icon name={'checkmark-circle'} style={{ color: 'green' }} />}
-            </Item>
-          </Form>
-
-          {this.state.showSignUp ?
-            <Button
-              style={{ padding: 5 }}
-              onPress={() => this.setState({ showSignUp: false })}
-              transparent
-            >
-              <Text>Already registered?</Text>
-            </Button> :
-            <Button
-              style={{ padding: 5 }}
-              onPress={() => this.setState({ showSignUp: true })}
-              transparent
-            >
-              <Text>{'Don\'t have an account?'}</Text>
-            </Button>
-          }
-
-          {this.state.showSignUp ?
-            <Button disabled={this.state.password.length < 6} onPress={this.signup}>
-              <Text>Sign up</Text>
-            </Button> :
-            <Button disabled={this.state.password.length < 6} onPress={this.signin}>
-              <Text>Sign in</Text>
-            </Button>
-          }
-
+          <View style={{ width: 350 }}>
+            <Form style={{}}>
+              <Item style={styles.marginBottom} regular>
+                <Input
+                  ref={(component) => { this._emailInput = component; }}
+                  onChangeText={(text) => { this.setState({ email: text }); }}
+                  placeholder='Email'
+                  autoCapitalize='none'
+                  value={this.state.email}
+                />
+                {/.+@.+\..+/i.test(this.state.email) && <Icon name={'checkmark-circle'} style={{ color: 'green' }} />}
+              </Item>
+              <Item regular>
+                <Input
+                  ref={(component) => { this._passwordInput = component; }}
+                  onChangeText={(text) => { this.setState({ password: text }); }}
+                  placeholder='Password'
```

Comment:
I wonder if this placeholder shows if the value is an empty string? Will verify after merge.